clipmenu

```diff
@@ -58,6 +58,8 @@ else
     chosen_line=$(list_clips | "$CM_LAUNCHER" "${launcher_args[@]}" "$@")
 fi
 
+launcher_exit=$?
```

Comment:
That's not quite right in the case at line 52. Shouldn't this be in the `else` block?

Reply:
If it's in the else block, and the variable on the last line is quoted, the last line would become `exit ""`, which causes an error `exit: numeric argument required`.

So, either this can't be in the else block, or the variable on the last line shouldn't be quoted (if it's not quoted, the line becomes `exit `, which is fine). I'm fine with doing it either way. :slightly_smiling_face: 

Yet another option would be to put the variable in the else block, and at the end check if the variable is empty, and only exit with it in that case?

Reply:
Moved the variable under the else statement.